overlay.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, bike, width=1280, height=740, bg: str = None):
         self.client.on_connect = self._on_connect
         self.client.on_disconnect = self.on_disconnect
         self.client.on_log = self.on_log
+        self.client.is_connected = False  # Default is false until connected
```

Comment:
Can you set `self.client.reconnect_delay_set(max_delay=10)` or something like that? By default, the MQTT client can wait up to 2 minutes before retrying connection which is a bit long. I don't think there's anything to loose by trying more regularly, it's not like we have hundreds of devices on our network all retrying